src/test/ui/const-generics/const_evaluatable_checked/nested-abstract-consts-1.rs

```diff
@@ -0,0 +1,22 @@
+// run-pass
```

Comment:
Please leave a comment in each of the tests giving a quick human readable explanation of what the test is testing